tslib.es6.js

```diff
@@ -348,6 +348,23 @@ export function __disposeResources(env) {
     return next();
 }
 
+export function __rewriteRelativeImportExtension(path, preserveJsx) {
+    if (typeof path === "string" && path[0] === "." && (path[1] === "/" || path[1] === "." && path[2] === "/")) {

```

Comment:
IIRC, string indexing has compatibility issues with older browsers (notably, IE7), so `.charAt` is usually safer to use. IIRC, we don't support IE7 anymore, so it probably won't matter.